### ISSUE TITLE
ФСЗН, пенсионный, расходы по перечислению на карт счет

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/GedeminSalary#104]
Эти суммы зависят от зарплаты работника, но выплачивает их предприятие. Надо их показывать на экране, отдельным разделом (не закладкой, а разделом, как, например, вычеты).

**Comment by alexandra-gsoftware** [comment]
расходы по перечислению на карт счет  - что это? Такого начисления нет в стандарте

**Comment by YRozmysl** [comment]
Мне кажется, что такие налоги, которые платит предприятие (ФСЗН, Белгосстрах) не нужно отражать в расчетном листке. тем более, что в явном виде(сумма) они хранятся только в проводках, и больше нигде.  Расходы по перечислению на карт счет тоже не совсем понятно, что имеется ввиду.

**Comment by gsbelarus** [comment]
Нет. Их надо показывать. Отдельный раздел наверное нужен -- налоги выплачиваемые предприятием.

так же и расходы связанные с перечислением на карт счет, которые предприятие несет.

**Comment by YRozmysl** [comment]
Вы уверены? Просто я не совсем понимаю для чего видеть и ЗНАТЬ сотруднику, что предприятие выплатило  ФСЗН на сумму 3,5 млрд. Ведь таким образом он может запросто рассчитать, какой вообще фонд заработной платы на предприятии, что я является коммерческой тайной.

**Comment by gsbelarus** [comment]
Не по всему предприятию, а по сотруднику. Хотя бы справочно, если нет такой аналитики в бд.

**Comment by YRozmysl** [comment]
Тогда можно попробовать  взять данные по сотруднику за нужный нам месяц из таблицы USR$WG_SOCINSURANCEINFO (Информация по пенсионным взносам)
сумма по человеку за месяц из поля USR$SUM * % ФСЗН  - это и будет фсзн по человеку,а
сумма по человеку за месяц из поля USR$ALLSUM* %Белгосстрах - это Белгосстрах по человеку.
А вот затраты по переводу, мне кажется, точно не нужны.
